src/components/NcBreadcrumbs/NcBreadcrumbs.vue

```diff
@@ -573,7 +572,6 @@ export default {
 					props: {
 						href,
 						title,
-						name: '', // TODO: Remove this once nameTitleFallback is removed from actionText.js mixin
```

Comment:
Shouldn't this be passing on the name, like:
```
						name,
```

So that name is set in NcActionLink and NcActionRouter elements?

Reply:
No, the `name` prop of `NcBreadcrumb` does not map to the `name` prop of the `NcActions*`, it goes into the default slot of the actions, as we do it some lines below.
We could of course change that, and map `breadcrumb:name`->`actionLink:name` and `breadcrumb:default-slot` -> `actionLink:default-slot`, but then we would also need to introduce an actions slot for `NcBreadcrumb`, since the actions currently go into the default slot. But I would say that this is out of scope for this PR here and something for a follow-up, if we decide to do that.

Reply:
Ah, was not aware.
No, don't need to add more stuff to it. When it worked the same before we can just continue